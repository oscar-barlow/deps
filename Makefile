.PHONY: install
install: update
	cat aptfile.txt | grep -v '#' | xargs sudo apt install --yes
	sudo apt autoremove --yes

.PHONY: update
update:
	sudo apt-get update --yes

.PHONY: upgrade
upgrade:
	sudo apt-get upgrade --yes

.PHONY: snap-install
install: refresh
	cat snapfile.txt | xargs sudo snap install

.PHONY: snap-refresh
refresh:
	sudo snap refresh
