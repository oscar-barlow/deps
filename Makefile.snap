.PHONY: install
install: refresh
	cat snapfile.txt | xargs sudo snap install

.PHONY: refresh
refresh:
	sudo snap refresh
